server/errorHandler.ts

```diff
@@ -3,14 +3,15 @@
 import type { NextFunction, Request, Response } from 'express'
 import type { HTTPError } from 'superagent'
 import logger from '../logger'
+import paths from './paths/temporary-accommodation/static'
 
 export default function createErrorHandler(production: boolean) {
   return (error: HTTPError, req: Request, res: Response, next: NextFunction): void => {
     logger.error(`Error handling request for '${req.originalUrl}', user '${res.locals.user?.username}'`, error)
 
     if (error.status === 401 || error.status === 403) {
-      logger.info('Logging user out')
-      return res.redirect('/sign-out')
+      logger.info('user not authorised')
```

Comment:
Capitalise first letter to keep consistency with `logger.info` messages using sentence case